core/embed/rust/src/ui/model_tr/component/button.rs

```diff
@@ -465,6 +503,14 @@ impl<T> ButtonDetails<T> {
         self
     }
 
+    /// Second icon for the pressed state
+    pub fn with_pressed_icon(mut self, icon: Option<Icon>) -> Self {
```

Comment:
it doesn't seem that this belongs in this PR?

Reply:
using this for proper inversion of some button icons - e.g. when it has added one column of pixels to satisfy even width requirement, you don't want to invert the extra column when pressed, hence the need for second icon. 

though @grdddj said that he was solving something related to this, so maybe it isnt needed anymore

Reply:
but i'm unable to lookup usages in the bootloader code because it's hidden like three layers of indirection away from actual callers :woman_shrugging:

Reply:
thats true, the `button_controller` and `button` components are implemented that way

Reply:
Deleted in [a59c096](https://github.com/trezor/trezor-firmware/pull/2959/commits/a59c096a27d4d5ec653b1176d63ea95945e20a2f) - now we can invert even odd-wide icons precisely